src/microbe_stage/MicrobeHUD.cs

```diff
@@ -625,22 +625,25 @@ private void UpdateHoverInfo(float delta)
             // Create for each compound the information in GUI
             foreach (var entry in compounds)
             {
-                var hBox = new HBoxContainer();
-                var compoundName = new Label();
-                var compoundValue = new Label();
+                if (entry.Value >= 0.1)
```

Comment:
I think hhyyrylainen mentioned somewhere that early returns are preferred to not have extra indentations